UPGRADE-3.x.md

```diff
@@ -9,3 +9,7 @@ UPGRADE FROM 3.0 to 3.1
 All files under the ``Tests`` directory are now correctly handled as internal test classes. 
 You can't extend them anymore, because they are only loaded when running internal tests. 
 More information can be found in the [composer docs](https://getcomposer.org/doc/04-schema.md#autoload-dev).
+
+### Deprecated
+
+`$container` property in `Security/SessionDownloadStrategy` is deprecated. Use `SessionInterface` `$session` instead.
```

Comment:
Looks ok but I'm not an expert of media-bundle so I don't know if the example is enough.

cc @sonata-project/contributors for review.